pages/satistics.js

```diff
@@ -186,54 +201,111 @@ export default function Satistics ({ ssrData }) {
     const inc = new Set(router.query.inc?.split(','))
     return inc.has(filter)
   }
-
   const { walletHistory: { facts, cursor } } = data || ssrData
+  const totalStacked = userStatsIncomingSats[0].data?.reduce((acc, d) => acc + d.value, 0)
+  const totalSpent = userStatsOutgoingSats[0].data?.reduce((acc, d) => acc + d.value, 0)
+  const totalEngagement = userStatsActions[0].data?.reduce((acc, d) => acc + d.value, 0)
```

Comment:
I think this is right. I'm getting the most recent item and summing them all together. But let me know if I'm thinking about the data wrong